lib/media/mp4_vtt_parser.js

```diff
@@ -34,10 +34,12 @@ goog.require('shaka.util.TextParser');
  * @param {number} offset
  * @param {?number} segmentStartTime
  * @param {?number} segmentEndTime
+ * @param {?boolean} useRelativeCueTimestamps
```

Comment:
Drop "?" and add a comment that useRelativeCueTimestamps is only used by the VTT parser.